pkgs/development/tools/java/jprofiler/default.nix

```
@@ -1,45 +1,45 @@
 { stdenv
 , lib
+, callPackages
 , fetchurl
 , makeWrapper
 , makeDesktopItem
 , copyDesktopItems
-, undmg
+, autoPatchelfHook
+, callPackage
 , jdk
+, libX11
+, component ? "ui"
 }:
 
 let
-  inherit (stdenv.hostPlatform) system;
-  pname = "jprofiler";
+  options = import ./version.nix;
```

Maybe you should name it something like this for clarity.
```suggestion
  versions = import ./versions.nix;
```

---

I not sure that `versions` is better naming. But maybe it is.